### PR TITLE
Add support for applying a target flash to actors through lua

### PR DESCRIPTION
**Pull request by abcdefg30**
Allows to code stuff like https://youtu.be/afzfD0AGTPQ?t=5m32s.

### CODE REVIEW

File: OpenRA.Mods.Common/Scripting/Properties/GeneralProperties.cs [hunk 1]
```diff
@@ -72,6 +73,13 @@ public bool HasProperty(string name)
 		{
 			return Self.HasScriptProperty(name);
 		}
+
+		[Desc("Render a target flash on the actor. If set, 'asPlayer'",
```
Comment (Phrohdoh) [inline OpenRA.Mods.Common/Scripting/Properties/GeneralProperties.cs]: Are these strings joined together with a space or directly concatenated?
If the latter you'll want to insert a space at the end of the first string or at the beginning of the second.

Comment (abcdefg30) [inline OpenRA.Mods.Common/Scripting/Properties/GeneralProperties.cs]: They should be two different lines, iirc (as in ", == \n").

File: OpenRA.Mods.Common/Scripting/Properties/GeneralProperties.cs [hunk 2]
```diff
@@ -72,6 +73,13 @@ public bool HasProperty(string name)
...
+			"defines which player palette to use. Duration is in ticks.")]
+		public void FlashTarget(int duration = 4, Player asPlayer = null)
```
Comment (pchote) [inline OpenRA.Mods.Common/Scripting/Properties/GeneralProperties.cs]: This doesn't accept a target argument\ to flash, so "FlashTarget" isn't a great name.  How about just "Flash"?

**Review by pchote — CHANGES_REQUESTED** [review]

**Comment by abcdefg30** [comment]
Updated.

**Review by pchote — APPROVED** [review]
Works as advertised, and seems like a reasonable API addition to make.

**Comment by reaperrr** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/9a804f47725982845d9633d5b6fd92c1335ad5ae...66976c0cd3f116f3ed068b1581edd12f725e9903)

**Pull request merged**